tools/gen_nvfuser_version.py

```diff
@@ -11,14 +11,18 @@
 
 # note that this root currently is still part of pytorch.
 def get_sha() -> str:
+    import os
+
+    # assume the $NVFUSER_VERSION is in sha form
+    nvfuser_version = os.environ.get("NVFUSER_VERSION")
```

Comment:
nitpick: It's probably just a bad variable name that we used in our script. But using `NVFUSER_VERSION` as sha sounds strange...

Is this just the timestamp that we attach in our nightly release? should we rename this maybe?

Reply:
I think an alternative is in the `get_version()` function below, when `get_sha()` returns UNKNOWN, we can use the environment variable in its absence. That's probably an easier justification.

Reply:
Agree that the variable name is very confusing to assume an sha format. The environment we have is that git will not work, but NVFUSER_VERSION (this could probably be renamed) inherited the approximate SHA from build pipeline.